fdbserver/ptxn/TLogPeekCursor.actor.cpp

```diff
@@ -251,7 +251,7 @@ StorageTeamPeekCursor::StorageTeamPeekCursor(const Version& beginVersion_,
   : VersionSubsequencePeekCursorBase(), storageTeamID(storageTeamID_), pTLogInterfaces(pTLogInterfaces_),
     pAttachArena(pArena_), deserializer(emptyCursorHeader(), /* reportEmptyVersion_ */ true),
     wrappedDeserializerIter(deserializer.begin(), pArena_), beginVersion(beginVersion_),
-    reportEmptyVersion(reportEmptyVersion_), lastVersion(beginVersion_ - 1) {
+    reportEmptyVersion(reportEmptyVersion_), rpcVersion(beginVersion_) {
```

Comment:
can you explain why we previously -1, and now we do not , please?

Reply:
The reason for introducing minus one previously here is because:

1. Previously our lastVersion is inclusive, which means, if TLog sends versions 3, 5, 7, 9, lastVersion will be 9 and we will start with version 10, or lastVersion + 1. Since we want to start with beginVersion, lastVersion has to be beginVersion - 1.
2. Now lastVersion is exclusive, which means we should start with lastVersion in the next RPC call. Hence rpcVersion should be the same as beginVersion.